medicines/web/components/drug-index/index.tsx

```diff
@@ -122,8 +122,12 @@ const DrugIndex: React.FC<IIndex> = ({ title, items, horizontal }) => {
               <li key={item.name} className={level > 0 ? 'substance-name' : ''}>
                 <Link
                   href={`?${
-                    level < 2 ? 'substance' : 'page=1&search'
-                  }=${encodeURIComponent(item.name)}`}
+                    level < 2 ? 'substance' : '&product=true&page=1&search'
```

Comment:
I think this is getting into the realms of extracting to a function.

Reply:
pulled it out in 9952d1a